src/components/Sections/Topics/TopicContainer.js

```diff
@@ -107,7 +107,7 @@ const TopicContainer = ({ t }) => {
   const dispatch = useDispatch();
 
   useEffect(() => {
-    dispatch(actions.fetchStats(namespace, {}, true));
+    dispatch(actions.fetchStats(namespace, {}, { isPrepare: true, countC: true, countL: true }));
```

Comment:
What is this {} action? What is it for?

Reply:
second param - base params - for example for topic page it will _tag=[tag uid]_